README.md

```diff
@@ -35,6 +35,7 @@ export interface Options {
   },
   target?: string
   format?: string
+  jsx?: 'transform' | 'preserve' | 'automatic'
```

Comment:
Exactly what I'm looking for. Thanks for this change, I'd love it to be reviewed by the project owner.